test/webRDFS/WebServerTest.cc

```diff
@@ -45,6 +45,44 @@ TEST(WebServerTest, testRead) {
   system("hdfs dfs -fs hdfs://localhost:5351 -rm /fileToRead");
 }
 
+TEST(WebServerTest, testCreate) {
+  system(
+        "hdfs dfs -fs hdfs://localhost:5351 "
+        "-copyFromLocal fileForTesting /fileToCreate");
+
+      ASSERT_EQ(0,
+                system("curl -i --insecure https://localhost:8080/webhdfs/v1/"
```

Comment:
Do we have server-side logic to handle writing the multipart form data passed by curl via `-T <filename>` to the newly created file?